src/frontends/lean/cmd_table.h

```diff
@@ -25,7 +25,7 @@ struct cmd_meta {
              optional<std::string> const & doc = optional<std::string>()):
         m_attrs(attrs), m_modifiers(mods), m_doc_string(doc) {}
 
-    void throw_exception_if_nonempty() const & {
+    void throw_exception_if_nonempty() const {
```

Comment:
I slipped in this change, correcting my last PR, since `const &`, while it works, is odd since it constrains this function to only work for lvalues.